### DISCUSSION TITLE
Install Slack GitHub application

### DISCUSSION THREAD

**Discussion by vprivat-ads** [stac-utils/.github#5] [Ideas]
We would like to be notified of new releases of several projects (pgstac, stac-fastapi, etc.) directly in our Slack instance, would it be possible to install the [Slack GitHub app](https://github.com/apps/slack/) in this organization?

**Comment by gadomski** [comment]
🤔 looking through the Slack+Github application, it looks like the permissions need to be configured on a per-Slack-workspace level. If we were to enable this integration, I think we'd want it to be to cloudnativegeo.slack.com. I'm not against it per-se, but I think there's lower-impact ways to get notified of new releases:

- Subscribe to the Github releases atom feed, e.g. https://github.com/stac-utils/stac-fastapi/releases.atom
- Subscribe to the PyPI feed, e.g. https://pypi.org/rss/project/pypgstac/releases.xml
